study.md

```diff
@@ -22,7 +22,9 @@ In your own words, explain one reason why we use "this". In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+A reason to use t..his is when a function uses the syntax, the user will be able to use the .call to bound an object.
```

Comment:
The important thing to remember for now is that when we call a method on an object, `this` in the method will refer to the object.